app/views/static_pages/home.html.erb

```diff
@@ -5,7 +5,9 @@
     <header>
       <h1 class="accent hero__main-heading"> Learn Web Development for Free </h1>
       <p class="secondary hero__sub-heading"> Stop wasting time...&nbsp;  Here's what you need to know. </p>
-      <%= link_to 'Sign Up', sign_up_path(ref: 'homenav'), class: 'button button--primary' %>
+      <% if current_user.nil? %>
```

Comment:
I'm not a fan of `.nil?` checks, it usually means theres logic missing elsewhere. I also think the heading will look a bit odd without a button of some description in it